src/nethsec/firewall/__init__.py

```diff
@@ -1177,6 +1177,22 @@ def is_output_rule(rule: dict) -> bool:
         return True
     return False
 
+def is_zone(uci, name:str) -> bool:
+    """
+    Check if name is a zone
+
+    Args:
+        name: zone to check
+
+    Returns:
+        True if name is a zone, False otherwise
+    """
+    zones = list_zones(uci)
+    for key, value in zones.items():
```

Comment:
The key var is not used:
```suggestion
    for value in zones.items():
```